juju/js-libjuju
# Update juju 3.0 beta4

## Description

- Add missing facades for Juju 3.0 beta4
- Group facade list by facade name in README

### QA

- Checkout this PR
- In the root of the PR codebase: `npm install && npm run build && yarn link`
- Install a beta 3.0 of Juju: `snap install juju --channel=3.0/beta`
- Make sure that your Juju instance is working: `juju dashboard`
- Clone the jaas-dasbhoard: `git clone https://github.com/canonical/jaas-dashboard`
- Add a new file at `public/config.local.js`:
```js
var jujuDashboardConfig = {
  controllerAPIEndpoint: "ws://<your-dashboard-host:port>/api",
  baseAppURL: "/",
  identityProviderAvailable: false,
  identityProviderURL: "",
  isJuju: true,
};
```
- In the root of `jaas-dashboard` run: `yarn && yarn link @canonical/jujulib`
- Run the dasbhoard: `yarn start`
- Go to localhost:3000
- Make sure that the dashboard is working without errors

## Issues

Fixes https://github.com/canonical/marketplace-tribe/issues/2882
